### ISSUE TITLE
Поддержка какого набора?

### ISSUE THREAD

**Issue by majioa** [slavonic/cu-tex#28]
Здравствуйте,

тут планируется поддержка какого набора? Unicode, или hip, или обоих?

**Comment by pgmmpk** [comment]
В планах только Unicode.  А почему интерес к HIP? Наверное можно его поддержать тоже, вытащив нужные куски из hiptex, но сначала будет сделан Unicode.

**Comment by majioa** [comment]
Интерес к hip-у таков, что некоторые символы из его набора, напр в глаголице, или в его дополнениях даже не планируются быть размещёнными в unicode.

**Comment by pgmmpk** [comment]
Спасибо за информацию.

@typiconman Александр, а каков статус глаголицы в Unicode? Вроде было
подготовлено предложение (http://www.ponomar.net/files/glagolitic.pdf)?

2016-04-03 16:49 GMT-04:00 Павел Скрылёв (Malo Skrylevo) <
notifications@github.com>:

**Comment by typiconman** [comment]
@pgmmpk Заявка была принята и выносные буквы глоголицы будут в девятой версии Юникода. В нашем шрифте Menaion Unicode они уже есть.

@majioa Какие символы Вам нужны для набора? Можно сделать доступ к PUA через разметку.

**Comment by majioa** [comment]
@typiconman ну, скажем, то что я видел в поддержке глаголицы, там есть какой-то символ, который оказывается спорным (нужно смореть какой), потом, есть и символы которые просто там не учтени, хотя и встречаются в хорватских и болгарских рукописях, типа широкого "она", и т.п. 

А ещё есть в одном из расширений хипа, поддержка крюковой азбуки, вместо обычных надстрочников.

**Comment by typiconman** [comment]
@majioa По поводу дополнительных символов глаголицы, напр. широкого "она", вопрос не исчерпан. Но нужны конкретные примеры. Те примеры, которые предоставлял Х. Миклас Технический комитет не нашел убедительными. Предлагаю Вам составить перечень того, что нужно для хорватских и болгарских рукописей, и мы на первом этапе добавим это в PUA. Потом можно написать и заявку в Технический комитет. Но мне сложно этим заниматься -- я не специалист в области глаголицы.

Что касается музыкальной нотации, в том числе и крюковой, то она, увы, далеко за пределами того, что мы пытаемся делать в этом пакете. Здесь мы пока только разрабатываем TeX шаблоны и утилиты для верстки богослужебных (ненотированных) текстов, а также для работы с ЦС текстом в научных (русско-, англо-, и т.п. язычных) публикациях.

**Comment by majioa** [comment]
> Предлагаю Вам составить перечень того, что нужно для хорватских и болгарских рукописей, и мы на первом этапе добавим это в PUA.

Ну постараюсь, как только будет для неё свободное время.

> Что касается музыкальной нотации, в том числе и крюковой, то она, увы, далеко за пределами того, что мы пытаемся делать в этом пакете.

Дело в том, что те значки ударений, придыханий и возгласов это тоже свое образная музыкальная нотация, только более простая и широкораспространённая. Так что, что включать в поддержку пакета или не вклчать, решать именно и только вам...

> а также для работы с ЦС текстом в научных (русско-, англо-, и т.п. язычных) публикациях.

тут как раз hip-бы и пригодился, хотябы вследстиве того, что мы сами комитет определяющий оный стандарт. =)

**Comment by typiconman** [comment]
Ой, давайте не будем заводить здесь спор Юникод vs. HIP. Каждый пользуется тем, что ему нравится. Ведь для работы с HIP есть же hiptex? Или там чего-то не хватает? Я не знаю, т.к. сам с HIP'ом не работаю.

**Comment by majioa** [comment]
> Ведь для работы с HIP есть же hiptex? Или там чего-то не хватает?

Не хватает, он не поддерживает уникодные символы кириллицы в документах, а переделать с наскока у меня не вышло, то есть я понял, что увязну и ничего толком не учиню.

**Comment by typiconman** [comment]
> он не поддерживает уникодные символы кириллицы в документах

Т.е. Вы хотите работать с HIP'ом в Юникоде (точнее, в UTF-8)?

**Comment by majioa** [comment]
> Т.е. Вы хотите работать с HIP'ом в Юникоде (точнее, в UTF-8)?

Да, конечно в `utf-8`, нынешний линукс уже мало который koi-8, и тем более cp1251 поддерживает, в основном на уровне почты и ядра.

**Comment by typiconman** [comment]
Enhancement request: add support for HIP, e.g., as:
\begin{hip}
Some text in HIP
\end{hip}
